scala/private/rule_impls.bzl

```diff
@@ -122,12 +122,14 @@ def _collect_plugin_paths(plugins):
     for p in plugins:
         if hasattr(p, "path"):
             paths.append(p)
+        elif p[JavaInfo] and p[JavaInfo].full_compile_jars:
+            paths.extend(p[JavaInfo].full_compile_jars.to_list())
         elif hasattr(p, "scala"):
             paths.append(p.scala.outputs.jar)
         elif hasattr(p, "java"):
             paths.extend([j.class_jar for j in p.java.outputs.jars])
-        # support http_file pointed at a jar. http_jar uses ijar,
-        # which breaks scala macros
+            # support http_file pointed at a jar. http_jar uses ijar,
+            # which breaks scala macros
```

Comment:
this comment is on the elif, can we move it below or unindent? It feels awkward to me being indented but not related to the current block.